CAIPs/caip-275.md

```diff
@@ -94,11 +94,11 @@ The Authentication flows definition JSON MUST conform to the following [Draft 7
   "properties": {
     "address": {
       "type": "string",
-      "description": "The address requested by the user. This can be from any chain."
+      "description": "The wallet address requested by the user. This can be from any chain. And can be an externally owned account or a smart wallet"
```

Comment:
```suggestion
      "description": "The wallet address requested by the user. This can be from any chain and can be an externally owned account or smart wallet"
```